tests/models/gpt_neox/test_modeling_gpt_neox.py

```diff
@@ -253,7 +253,7 @@ class GPTNeoXModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMi
 
     def setUp(self):
         self.model_tester = GPTNeoXModelTester(self)
-        self.config_tester = ConfigTester(self, config_class=GPTNeoXConfig, hidden_size=37)
+        self.config_tester = ConfigTester(self, config_class=GPTNeoXConfig, hidden_size=64, num_attention_heads=8)
```

Comment:
8 attention heads is pretty large for a test. Could we make this 4?

Reply:
This is only used for `run_common_tests`, which does not run any forward passes or anything